docs/development.rst

```diff
@@ -30,6 +32,8 @@ Build Instructions
                   
     $ env PATH=$HOME/Library/Python/2.7/bin/:"$PATH" cmake -G Ninja -DCMAKE_BUILD_TYPE:=Debug ..
 
+Note: this assumes you have installed `lit` as a user, by running `easy_install --user lit`.
```

Comment:
The Xcode project will automatically do this for you, shame the CMake doesn't. :/